template_content/smart_contracts/config.py.jinja

```diff
@@ -14,14 +13,15 @@ from algokit_utils import (
 from algosdk.util import algos_to_microalgos
 from algosdk.v2client.algod import AlgodClient
 from algosdk.v2client.indexer import IndexerClient
-{% endif %}
```

Comment:
Need to keep this endif and the other if that was one line 24, as this file can also be present when using a typescript deployment

Reply:
due to the number of if statements in this file, it might actually be easier to have two different config.py with different content based on the deployment_language setting e.g.

`{% if deployment_language == 'python' %}config.py{% endif %}`
Has build + deploy config

`{% if deployment_language == 'typescript' %}config.py{% endif %}`
Only has build config